AirCasting/ABConnector/SDSyncController.swift

```diff
@@ -67,20 +67,21 @@ class SDSyncController {
                 case .success(let metadata):
                     Log.info("[SD SYNC] Finished reading data with success")
                     progress(.finalizing)
-                    let files = self.fileWriter.finishAndSave()
-                    Log.info("[SD SYNC] Files: \(files)")
-                    guard !files.isEmpty else {
+                    let directories = self.fileWriter.finishAndSave()
+                    Log.info("[SD SYNC] Files: \(directories)")
+                    guard !directories.isEmpty else {
                         Log.info("[SD SYNC] No files. Finishing sd sync")
+                        Log.info("### completion success called in line 75")
                         completion(.success(()))
                         return
                     }
                     
                     // MARK: checking if files have the right number of rows and if rows have the right values
-                    self.checkFilesForCorruption(files, expectedMeasurementsCount: metadata.expectedMeasurementsCount) { fileValidationResult in
+                    self.checkFilesForCorruption(directories, expectedMeasurementsCount: metadata.expectedMeasurementsCount) { fileValidationResult in
                         switch fileValidationResult {
                         case .success(let verifiedFiles):
```

Comment:
I think we need to adjust the naming here to mimic the new filesystem scheme (so directories here instead of files)